### PR TITLE
Port to Minecraft 1.21

### PR DESCRIPTION
**Pull request by IMS212**

### CODE REVIEW

File: src/main/java/dev/lambdaurora/spruceui/util/RenderUtil.java [hunk 1]
```diff
@@ -50,29 +52,7 @@ public static void renderBackgroundTexture(int x, int y, int width, int height,
...
-				.uv(right / 32.f, bottom / 32.f + vOffset)
-				.color(red, green, blue, alpha).next();
-		bufferBuilder.vertex(right, y, 0)
-				.uv(right / 32.f, y / 32.f + vOffset)
-				.color(red, green, blue, alpha).next();
-		bufferBuilder.vertex(x, y, 0)
-				.uv(0, y / 32.f + vOffset)
-				.color(red, green, blue, alpha).next();
-		tessellator.draw();
+
```
Comment (LambdAurora) [inline src/main/java/dev/lambdaurora/spruceui/util/RenderUtil.java:55]: Render background gone???

**Review by LambdAurora — COMMENTED** [review]
omg you're so based for this.

Also sorry for inflicting you this pain.
And what the heck is DeltaTracker??? Oh god I am scared now :sob:

File: src/main/java/dev/lambdaurora/spruceui/util/RenderUtil.java (hunk 1 above)
Comment (IMS212) [reply src/main/java/dev/lambdaurora/spruceui/util/RenderUtil.java:55]: wasn't sure what you wanted to replace it with

Comment (IMS212) [reply src/main/java/dev/lambdaurora/spruceui/util/RenderUtil.java:55]: (also feel free to commit anything to the PR)

Comment (LambdAurora) [reply src/main/java/dev/lambdaurora/spruceui/util/RenderUtil.java:55]: > wasn't sure what you wanted to replace it with

well I'm cooking an implementation of a proper replacement for 1.20.6 then I'll do a proper merge of that

**Pull request closed without merging**